evaluation/metrics/metrics.py

```diff
@@ -20,6 +21,83 @@
 
 
 # eval_metrics, take predictions and targets as input
+def get_dataperf_accuracy(predictions: list, targets: list):
+    """
+    Here, predictions can be a list of lists of acceptable multilabel lists, instead
+    of just a list of multilabel lists. This is helpful for stochastic models, where we
+    also want to report a standard deviation over the model outputs.
+    """
+
+    if not isinstance(predictions[0], list):
```

Comment:
Does each index in the predictions variable correspond to the predictions over the whole test set for a specific iteration?

Reply:
Nevermind, it looks like each index is predictions corresponds to a list of predictions for all iterations for a specific sample in the test set.